doc/lua_api.md

```diff
@@ -9541,7 +9289,7 @@ See [Decoration types]. Used by `minetest.register_decoration`.
     rotation = "90",
     -- Rotation can be "0", "90", "180", "270", or "random"
 
-    place_offset_y = 0,
+    place_offset_y = 0
```

Comment:
it's better to keep consistent commas here IMO

Reply:
I think they look tacky. New features aren't added to the API often, so keeping commas for easy additions is unnecessary. That's just my opinion though, if people object, I'll add them back.

Reply:
Trailing commas readded.